controllers/etcdcopybackupstask/reconciler.go

```diff
@@ -355,12 +355,39 @@ func (r *Reconciler) createJobObject(ctx context.Context, task *druidv1alpha1.Et
 							VolumeMounts:    volumeMounts,
 						},
 					},
-					Volumes: volumes,
+					ShareProcessNamespace: pointer.Bool(true),
+					Volumes:               volumes,
 				},
 			},
 		},
 	}
 
+	if r.Config.FeatureGates[string(features.UseEtcdWrapper)] {
+		if targetProvider == druidutils.Local {
+			// init container to change file permissions of the folders used as store to 65532 (nonroot)
+			// used only with local provider
+			job.Spec.Template.Spec.InitContainers = []corev1.Container{
+				{
+					Name:         "change-permissions",
```

Comment:
Please use consistent naming for the init containers across reconcilers.
```suggestion
					Name:         "change-backup-bucket-permissions",
```